packages/react-component-library/src/components/NumberInputE/NumberInputE.test.tsx

```diff
@@ -193,6 +193,31 @@ describe('NumberInputE', () => {
       assertOnChangeCall(123, 3)
     })
 
+    describe('and the user types a value with alpha characters', () => {
+      beforeEach(async () => {
+        const input = wrapper.getByTestId('number-input-input')
+
+        await userEvent.type(input, '1')
```

Comment:
FYI `userEvent.type` is typed to only return a promise if a delay is passed.

https://github.com/testing-library/user-event/blob/57a6ea4d4808bb5c07817089c437929ba9cdb070/src/type/index.ts#L4-L13

Is VS code wanting an `await` there?

Reply:
I was just following the pattern used in other tests and assumed it always needed to be awaited - I've updated this for all relevant tests now.